Source/Lib/pkg-config.pc.in

```diff
@@ -1,9 +1,9 @@
 prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
remove unused "prefix"?

Reply:
I would probably prefer to leave it for formality sake since most other pc files I've seen had prefix set, although they usually had `${prefix}` for the other parts so I might do that instead

Reply:
Sure. Thx.

Reply:
changed to use `${prefix}` for the rest of the dirs